solutions/6.js

```diff
@@ -0,0 +1,8 @@
+const callX2=(func,num,i=1)=>{
+  func(i);
```

Comment:
Double check the value you are passing into func.

i is the number of calls you have made, but that is not the value the function should use